api.md

```diff
@@ -779,8 +788,253 @@ Get all chunks older than 3 months and newer than 4 months:
 SELECT show_chunks(older_than => interval '3 months', newer_than => interval '4 months');
 ```
 
+## reorder_chunk() [](reorder_chunk)
+Reorder a single chunk's heap to follow the order of an index. This function
+acts similarly to the [PostgreSQL CLUSTER command] [postgres-cluster] , however
+it uses lower lock levels so that, unlike with the CLUSTER command,  the chunk
+and hypertable are able to be read for most of the process. It does use a bit
+more disk space during the operation. 
+
+This command can be particularly useful when data is often queried in an order
+different than it was originally inserted. For example, data is commonly
+inserted into a hypertable in loose time order (e.g., many devices concurrently
+sending their current state), but one might typically query the hypertable about
+a _specific_ device. In such cases, reordering a chunk using an index on
+device_id, time can lead to significant performance improvement for these types
```

Comment:
Some notation like `device_id, time` or (device_id, time) to show a composite index.